src/data/complete.queryMap.json

```diff
@@ -1132,7 +1130,6 @@
-  "15aa0a85a06807f72c1b64cbb3c312e8": "mutation updateConsignmentSubmissionMutation(\n  $input: UpdateSubmissionMutationInput!\n) {\n  updateConsignmentSubmission(input: $input) {\n    consignmentSubmission {\n      internalID\n    }\n  }\n}\n",
```

Comment:
@ashfurrow - I manually removed this, and a few others... but this seems wrong?

Reply:
yeah there should be no reason to delete entries from this file. It would degrade performance in the app, and the eigen deploy script would only add these entries back so we need to find out what the underlying problem is here.

Reply:
@ds300  - tests pass locally, so i think it has to do with the ENV bar above. If this doesn't break the app, lets go ahead and merge and then follow up once ENV var is flipped on in staging and so on.

Reply:
it will only get added back in the next time there's a build on eigen/master and then MP master will start failing. I think we should fix the env problem before merging this.

Reply:
Once it's merged into master and the ENABLE_CONSIGNMENTS_STITCHING var is flipped on in staging, the query map will regenerate and it will pass. 

@ds300 - You can test this locally by checking out this PR, checking out `src/data/complete.queryMap.json` from master, and then running `yarn jest`.

Reply:
Adding to CI worked 👍